### DISCUSSION TITLE
Weather description for weather app

### DISCUSSION THREAD

**Discussion by PizzaNCoding88** [open-meteo/open-meteo#767] [General]
Hello,

I am trying to create my own weather app with NextJS using your fantastic API, but I can't seem to find weather description for the day (like sunny, rainy, snowy, cloudy and so on).

Is this option available and I'm totally missing it?

If not available, I have noticed I can request weather codes, what would be the best way to convert them in weather description in JavaScript?

Thanks for any help/input

**Comment by patrick-zippenfenig** [comment]
Hi. You can use `weather code` and the look up table at the end of the [API documentation](https://open-meteo.com/en/docs). For JavaScript there is also a look up table available which is used for the interactive graph: https://github.com/open-meteo/open-meteo-website/blob/cb54698ae1677be707feca40979db3e1faac6735/src/routes/en/docs/ResultPreview.svelte#L194

**Reply by PizzaNCoding88** [reply]
Fantastic, thanks a lot.

With the list of codes I should be able to sort something out.
